### PR TITLE
Add note to build guide to flash blackpill before soldering

### PR DESCRIPTION
**Pull request by twalker**
I had a defective blackpill which threw errors upon flashing with QMK. 
Thankfully, I hadn't soldered it to the PCB yet, and had a spare. 
This adds a suggestion to the build guide to flash before soldering.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ To build the keyboard, you'll also need a soldering iron and some tin. Follow th
 
 ### Firmware
 
-The Cantor keyboard uses the QMK firmware. Firmware for this keyboard is comming soon to the main QMK repository with [this PR](https://github.com/qmk/qmk_firmware/pull/16552).
+The Cantor keyboard uses the QMK firmware. Firmware for this keyboard is coming soon to the main QMK repository with [this PR](https://github.com/qmk/qmk_firmware/pull/16552).
```
Comment (twalker) [inline README.md:60]: Unrelated spelling fix.

**Comment by fourstepper** [comment]
LGTM, although I am not the maintainer :D

**Comment by diepala** [comment]
Good point, although I also recommend using sockets for the microcontroller in case something goes wrong or the blackpill dies. Should we make this more clear in the build guide?
